patternfly/patternfly-elements
# Disclosure accordion variant

## Disclosure accordion variant

* Link to discovery documentation (should contain all the planning and requirements for this work):
   - See the `DISCOVERY.md` template at the root of the project
* Link(s) to demo pages where this element can be viewed:

---

### What has changed and why
* When an accordion has only one header/panel coupling it displays as a `disclosure`: the entire header has a 1px border when collapsed and the chevron moves to the left side of the header.

### Testing instructions
1. Go to the `/en/search` page
2. Enter a search string
3. The customer portal results at the top of the page should display as a disclosure rather than a standard accordion. You should see the chevron on the left side, the header should have a gray 1px border on all sides. And if you look in the inspector you should see the attribute `pfe-disclosure="true"` on the accordion itself, the header element and the panel element.

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
